packages/epo-react-lib/src/types/css.d.ts

```diff
@@ -0,0 +1,9 @@
+// My css.d.ts file
+import type * as CSS from "csstype";
+
+declare module "csstype" {
+  interface Properties {
+    // Allow any CSS Custom Properties
+    [index: `--${string}`]: any;
```

Comment:
slight modification to the typings for the `style` prop so we can pass CSS variables